src/dotnet/OpenAPI.Validator.Tests/OpenAPIModelerValidationTests.cs

```diff
@@ -699,6 +699,7 @@ public void CleanFileValidation()
                 MergeState = ServiceDefinitionDocumentState.Individual
             };
             var messages = this.ValidateOpenAPISpec(Path.Combine(AutoRest.Core.Utilities.Extensions.CodeBaseDirectory, "Resource", "OpenAPI", "Validation", "positive", "clean-complex-spec.json"), subtest1md);
+            
```

Comment:
Do you have any test case to cover this?

Reply:
Add unittest